### DISCUSSION TITLE
In 🇯🇵, there is an island called Okunoshima filled with what animal?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#710] [Galaxy Trivia] [answered]
**Bonus:** Use the animal emoji to describe its approximate headcount on the island.

**Comment by peterbaccaro** [comment]
Okunoshima, also known as Rabbit Island, is a small island located in the Inland Sea of Japan. As its nickname suggests, the island is famous for its large population of wild **rabbits**. 

The headcount is approximately **900** 🐰.

**Comment by SANKARAMDAS** [comment, marked as answer]
The Gateway to 🐰🐰🐰Rabbit Island. Official Site. OFF THE COAST OF HIROSHIMA, JAPAN. Located in eastern Hiroshima, Rabbit Island is a small enclave occupied by hundreds of wild rabbits that roam the forests and fields, chasing tourists for food.
The mysterious enclave of 1,000 🐰🐰off the coast of Hiroshima, Japan.

**Comment by thuongtruong1009** [comment]
In 🇯🇵, the island called Okunoshima is filled with 🐰 (rabbits). The approximate headcount of rabbits on the island is quite significant, so we can use the 🐰🐰🐰 (rabbit emoji repeated) to represent a large population of rabbits on Okunoshima.

**Comment by ParanoidUser** [comment]
**Extra Bonus:** What are the DOs/DON'Ts when visiting the island?

**Comment by SANKARAMDAS** [comment]
Don't - 
1. Don’t feed them any human foods or snacks.
2. Don’t feed them cabbage or lettuce. Although many tourist and blog sites say you can buy vegetables from the convenience store before you hop on the ferry, these leafy greens are extremely innutritious for rabbits and even harmful for their digestive system, often causing bloating.
3. Don’t pick them up, chase them, or try to force interaction with them. While they might look friendly, they can become aggressive if stressed.

D0-
 1. Poison Gas Museum
 2. Abandoned sites throughout the island with explanations
 3. Hot spring
 4. Camping
 5. Cycling
 6. Hiking
 7. Tennis
 8. Fishing
 
And so on...

**Reply by ParanoidUser** [reply]
My favorite is "DO NOT bring your own rabbits". 😄

**Reply by SANKARAMDAS** [reply]
🤣🤣
